src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2Serializer.cs

```diff
@@ -2054,7 +2054,7 @@ protected virtual void WriteProxyRestriction(XmlWriter writer, Saml2ProxyRestric
 
             // <Audience> - 0-OO
             foreach (Uri uri in proxyRestriction.Audiences)
-                writer.WriteElementString(Prefix, Saml2Constants.Elements.Audience, uri.OriginalString);
+                writer.WriteElementString(Saml2Constants.Elements.Audience, Saml2Constants.Namespace, uri.OriginalString);
```

Comment:
Other methods in this class are using this [overload ](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmlwriter.writeelementstring?view=net-5.0#System_Xml_XmlWriter_WriteElementString_System_String_System_String_System_String_System_String_)of WriteElementString. Curios why you didn't choose that overload.

```suggestion
                writer.WriteElementString(Prefix, Saml2Constants.Elements.Audience, Saml2Constants.Namespace, uri.OriginalString);
```

Reply:
@GeoK we could specify the ns, but it is not necessary as we are assuming the ns of the element as we use the same prefix. If one wanted to use a different ns, then it makes sense to specify it.